.size-limit.json

```diff
@@ -1,6 +1,6 @@
 [
   {
     "path": ["build/static/js/*.js", "build/static/css/*.css"],
-    "limit": "6 s"
+    "limit": "512 KB"
```

Comment:
In the future, I'd honestly like to see this go lower --- I think we can live with a ~100 KB limit for our public-facing code and 300-400 KB for all files (including vendor packages).

Reply:
For sure, I just wanted a conservative estimate initially.